sources/packages/backend/libs/sims-db/src/constant.ts

```diff
@@ -55,3 +55,5 @@ export const TableNames = {
   Messages: "notification_messages",
   Notifications: "notifications",
 };
+
+export const INSTITUTION_TYPE_BC_PRIVATE = 2;
```

Comment:
This constant looks more generic than a db constant, may be it can be put under `libs/utilities/system-configurations-constants.ts`

Reply:
mmm....for me... both files are okay. I added the constant here, as it is a database id